src/services/class.service.js

```diff
@@ -48,6 +48,20 @@ const deleteClassById = async (classId, teacher) => {
   return _class;
 };
 
+const getStudents = async (classId) => {
+  return Class.find({ _id: classId }).students;
```

Comment:
you probably need to populate this request in order to return not just ids but the objects